cves/CVE-2013-1966.yml

```diff
@@ -45,7 +45,7 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: URL requests containing parameterswere not properly handled, allowing malicious code in the native OGNL language to be executed through injection via the url parameters.
```

Comment:
Try to explain this in more layman's terms. For example, what does OGNL stand for?

Reply:
Updated with a definition of OGNL and a more simple explanation of XSS injection.